.travis.yml

```diff
@@ -0,0 +1,28 @@
+language: php
+
+sudo: false
+
+cache:
+  directories:
+    - "$HOME/.composer/cache"
+
+env:
+  - COMPOSER_FLAGS="--prefer-stable"
+
+php:
+  - 7.1
+
+matrix:
+  include:
+    - php: 7.1
+    - script: vendor/bin/phpcs -n
+      env: PHPCS_BUILD=1
+    - php: 7.1
```

Comment:
you have two php 7.1 builds, you might want to add `COMPOSER_FLAGS="--prefer-lowest"` to one so we test the lowest versions too.